lib/loc/loc.go

```diff
@@ -172,6 +172,11 @@ func (l Locator) String() string {
 	return str
 }
 
+// Human returns user-friendly string representation of the locator.
+func (l Locator) Human() string {
```

Comment:
I mean just `Human` carries a bit different connotation if you ask me, how about `Description` or `Readable`?